application/src/main/resources/db/V11__Add_Help_Channel_Plus_Creator.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE help_threads
```

Comment:
You gotta adjust the PP.md file (privacy policy) and add to it what we are logging extra now (associating user id with thread id and implicitly when and how many questions they ask) - for 30 days.